test-lit/lit_port.py

```diff
@@ -0,0 +1,256 @@
+""" lit_port.py
+
+    Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+    See https://llvm.org/LICENSE.txt for license information.
+    SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+"""
+
+import argparse
```

Comment:
Do we need all this import

Reply:
Sadly, all needed. Shlex is hardly used, but is preferable to str.split().